reference/forms/types/password.rst

```diff
@@ -122,3 +122,29 @@ The default value is ``''`` (the empty string).
 .. include:: /reference/forms/types/options/required.rst.inc
 
 .. include:: /reference/forms/types/options/row_attr.rst.inc
+
+Symfony UX TogglePassword Options
+---------------------------------
```

Comment:
Yup! I agree with Javier. So what we basically want to do is something like:

```rst
``toggle``
~~~~~~~~~~
**type**: ``boolean`` **requires**: `symfony/ux-toggle-password`_

Adds "Show"/"Hide" links to the field which toggle the password field to plaintext when clicked.
See `symfony/ux-toggle-password`_ for more details.
```

That's it! All the rest should be documented over on `symfony/ux`. The key thing here is to make this feature discoverable for someone that is browsing the official docs.